wageanalysis.sh

```diff
@@ -0,0 +1,14 @@
+#Returns highest earner, lowest earner, and number of females in top ten
+
+cat wages.csv|sed 's/,/ /g'|sort -k 4 -n|tail -n 1
+cat wages.csv|sed 's/,/ /g'|sort -k 4 -n|head -n 2|tail -n 1
+cat wages.csv|sed 's/,/ /g'|sort -k 4 -n|tail -n 10| grep female| wc -l
+<<<<<<< HEAD
```

Comment:
You need to delete these. They are references for merge conflicts and will cause an error when you run your script.

Reply:
Sure, but we wanted to make sure that you knew we worked separately and merged conflicts as mentioned in our exercise hand out!

Reply:
May I know why am I given 20% less?